dbms/src/Flash/Coprocessor/DAGQueryBlockInterpreter.cpp

```diff
@@ -258,20 +258,22 @@ void DAGQueryBlockInterpreter::handleJoin(const tipb::Join & join, DAGPipeline &
     size_t join_build_concurrency = settings.join_concurrent_build ? std::min(max_streams, build_pipeline.streams.size()) : 1;
 
     /// build side streams
-    executeExpression(build_pipeline, build_side_prepare_actions);
+    executeExpression(build_pipeline, build_side_prepare_actions, "Append Join key and join Filters");
```

Comment:
build side prepare actions?
or
Append Join key and join Filters for build side?